ceph/rados/core_workflows.py

```diff
@@ -4420,21 +4421,21 @@ def set_unmanaged_flag(self, daemon):
         log.error("The unmanaged flag is not set to True  ")
         return False
 
-    def set_managed_flag(self, daemon):
+    def set_managed_flag(self, service_type, service_name):
```

Comment:
same as above. Improve the method so that both older and newer calls work with minimal changes.